charts/visor/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.2.0
+version: 1.2.1
```

Comment:
Revision increment due to all new flags being optional in use, with defaults matching the app, and their use does not (intend to) change the default behavior.